presto-product-tests/src/main/resources/sql-tests/testcases/window_functions/windowDescriptionAABwithEP.sql

```diff
@@ -0,0 +1,8 @@
+-- database: presto; groups: window;
+select
```

Comment:
Extra Parameters (or Extended Play if you prefer your MP3s in analog format stored on big black slices of vinyl). Open to a better name that isn't super long.